tests/lax_numpy_test.py

```diff
@@ -1227,6 +1227,20 @@ def testPiecewise(self, shape, dtype, ncond, nfunc):
     self._CompileAndCheck(jnp_fun, args_maker, check_dtypes=True)
 
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+    {"testcase_name": "a_shape={}".format(
+      jtu.format_shape_dtype_string(a_shape, dtype)),
+     "dtype": dtype, "a_shape": a_shape}
+    for dtype in default_dtypes
+    for a_shape in one_dim_array_shapes))
+  def testTrimZeros(self, a_shape, dtype):
+    rng = jtu.rand_default(self.rng())
+    np_fun = lambda arg1: np.trim_zeros(arg1)
+    jnp_fun = lambda arg1: jnp.trim_zeros(arg1)
+    args_maker = lambda: [rng(a_shape, dtype)]
+    self._CheckAgainstNumpy(np_fun, jnp_fun, args_maker, check_dtypes=True)
```

Comment:
The test should also cover different options for the `trim` parameter

Reply:
Hi,

I have added the tests to cover all modes of the ```trim``` argument, but I just created different functions to achieve this. Is there a better way to do this using ```args_maker```.

Reply:
I think this would be cleaner to do via parametrization. See, for example, how the extra arguments are handled in `testUnique`: https://github.com/google/jax/blob/29aa9bfc8f5dbdf635e877395dc2082cefc8bc87/tests/lax_numpy_test.py#L1460-L1477

The benefit of the parametrized approach is that if one of the test cases fails, the other can still run. It gives a fuller picture of test coverage & results.

Reply:
We should test at least `"f"`, `"b"`, and `"fb"`, and maybe even an empty `trim` string.

Reply:
Hi,

I have attempted to add the parameterization for the trim argument. I have not included the empty ```trim``` argument test, as by default in ```numpy``` we have the value of ```trim``` set to ```fb```, which I have tried to maintain. Does this sound good?